src/main/java/com/odysseusinc/arachne/datanode/Constants.java

```diff
@@ -119,7 +119,8 @@
             String DELETE = "/api/v1/data-sources/{id}";
             String UPDATE = "/api/v1/data-sources/{id}";
             String CENTRAL_REGISTER = "/api/v1/data-sources/{id}/register-on-central";
-            String DS_MODEL_CHECK_RESULT = "/api/v1/data-sources/{id}/check/result/{password}";
+            String DS_MODEL_CHECK_FIRSTCHECK ="/api/v1/data-sources/{id}/check/result/{password}/firstcheck";
+            String DS_MODEL_CHECK_HEARTBEAT = "/api/v1/data-sources/{id}/check/result/{password}/heartbeat";
```

Comment:
Where it is used?

Reply:
Pls remove any related to heartbeat or healthcheck from public

Reply:
fixed